isb_web/main.py

```diff
@@ -102,7 +101,7 @@
 )
 app.mount(manage.MANAGE_PREFIX, manage_app)
 app.mount(debug.DEBUG_PREFIX, debug_app)
-app.mount(metrics.METRICS_PREFIX, metrics_app)
+app.include_router(metrics.router)
```

Comment:
Using this seems to work around the `root_path` problems where things were redirecting to the wrong place.

Reply:
Note that this seems like a cleaner approach overall that doing the embedded app approach.  We should probably convert the `manage_app` and `debug_app` implementations at some point.

Reply:
It seems that `include_router` is the way to work with an application composed of multiple files[^1] whereas `mount` is for combining independent applications[^2]. 

I think the `include_router` approach makes sense here since these are really parts of the same application, just on different routes, rather than separate applications.

[^1]: https://fastapi.tiangolo.com/tutorial/bigger-applications/
[^2]: https://fastapi.tiangolo.com/advanced/sub-applications/